core/src/main/java/hudson/model/Run.java

```diff
@@ -730,9 +730,12 @@ public String getDescription() {
      * Gets the string that says how long the build took to run.
      */
     public @Nonnull String getDurationString() {
-        if(isBuilding())
+        if (hasntStartedYet()) {
+            return Messages.Run_NotStartedYet();
+        } else if(isBuilding()) {
```

Comment:
Space fixed up. Thanks for the review.

On 25/03/2015 15:13, Kanstantsin Shautsou wrote:
> In core/src/main/java/hudson/model/Run.java
> <https://github.com/jenkinsci/jenkins/pull/1620#discussion_r27129951>:
> 
>> @@ -730,9 +730,12 @@ public String getDescription() {
>>       * Gets the string that says how long the build took to run.
>>       */
>>      public @Nonnull String getDurationString() {
>> -        if(isBuilding())
>> +        if (hasntStartedYet()) {
>> +            return Messages.Run_NotStartedYet();
>> +        } else if(isBuilding()) {
> 
> space after if please
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jenkinsci/jenkins/pull/1620/files#r27129951>.
>